lib/handlers/sender_handler.rb

```diff
@@ -158,62 +165,77 @@ def on_container_start(container)
         })
     end
 
+    def delay
+      before = @duration.delay("before-send")
+      after = @duration.delay("after-send") if @sent > 0 # No after-delay on first send
+      [before, after].compact.sum
```

Comment:
Array#sum is not supported in Ruby 2.0.0, I will change this to Array#inject(:+).